clickhouse_connect/driver/compression.py

```diff
@@ -0,0 +1,79 @@
+import zlib
+from abc import abstractmethod
+from typing import Union
+
+import lz4
+import lz4.frame
+import zstandard
+
+try:
+    import brotli
+except ImportError:
+    brotli = None
+
+
+def available_compression():
+    comp = ['lz4', 'zstd']
+    if brotli:
```

Comment:
basically, brotli will be not returned to the list until the package is available, that's correct?